### DISCUSSION TITLE
【要望】フォロワーリスト機能の廃止

### DISCUSSION THREAD

**Discussion by ko1nksm** [increments/qiita-discussions#900] [要望・アイデア・不具合]
ユーザーページに表示されているフォロワーリストは、現在スパムにしか利用されていないようです。

フォローリスト（ある人がフォローしているユーザーのリスト）を見ている人はいるかも知れませんが、
フォロワーリスト（ある人がフォローされた大半がスパム業者のリスト）を見ている人は誰もいないでしょう。

（無効にすることができる）フォローされた時の通知機能があれば十分だと思われます。
フォロワーリストの存在意義を思いつきません。

**Comment by tanaka-asami415** [comment]
@ko1nksm 
ご不便をおかけして申し訳ありません。

以前 https://github.com/increments/qiita-discussions/discussions/76 でご意見をいただいた時から、様々な対策等を行っておりますが
未だ、良くない体験となってしまっていて申し訳ありません🙇

運営としては、フォロワーリストを廃止にするのではなく
「スパム対策」「フォロワーリストがスパムで埋まってしまう問題」の改善をする方向で検討しております。

引き続き対策を検討していきますので、ご理解いただけますと幸いです。
